src/utils/mtev_skiplist.c

```diff
@@ -37,6 +37,8 @@ static int mtev_skiplisti_find_compare(mtev_skiplist *sl, const void *data,
                                        mtev_skiplist_node **prev,
                                        mtev_skiplist_node **next,
                                        mtev_skiplist_comparator_t comp);
+int mtev_skiplisti_remove(mtev_skiplist *sl, mtev_skiplist_node *m,
```

Comment:
I wondered about that... Since it was in the header file, it's possible that others were relying on the function... If we accept that possibility, then I think we'd want to bump the version to force downstream clients to re-link against the .so when the function goes away. Keeping it available but out of the header file strongly discourages new clients from using it, but shouldn't cause breakage in old clients that might.